project.clj

```diff
@@ -1,6 +1,6 @@
 (def ps-version "5.2.0-master-SNAPSHOT")
 (def jruby-1_7-version "1.7.27-1")
-(def jruby-9k-version "9.1.11.0-1")
+(def jruby-9k-version "9.1.15.0-2")
```

Comment:
Was this also intended as a combo update to 9.1.15?

Reply:
pretty much. i can break it into 2 PRs if that's easier. the jruby-utils releases were a little intermixed.

Reply:
In principle it's probably better in 2 PRs but right now whatever